doc/api/Buffer_Control/.docConfig.json

```diff
@@ -1,28 +1,35 @@
 {
-  "pages": [
-    {
-      "path": "./setWantedBufferAhead.md",
-      "displayName": "setWantedBufferAhead"
-    },
-    {
-      "path": "./getWantedBufferAhead.md",
-      "displayName": "getWantedBufferAhead"
-    },
-    {
-      "path": "./setMaxBufferBehind.md",
-      "displayName": "setMaxBufferBehind"
-    },
-    {
-      "path": "./getMaxBufferBehind.md",
-      "displayName": "getMaxBufferBehind"
-    },
-    {
-      "path": "./setMaxBufferAhead.md",
-      "displayName": "setMaxBufferAhead"
-    },
-    {
-      "path": "./getMaxBufferAhead.md",
-      "displayName": "getMaxBufferAhead"
-    }
-  ]
-}
+    "pages": [{
+            "path": "./setWantedBufferAhead.md",
+            "displayName": "setWantedBufferAhead"
+        },
+        {
+            "path": "./getWantedBufferAhead.md",
+            "displayName": "getWantedBufferAhead"
+        },
+        {
+            "path": "./setMaxBufferBehind.md",
+            "displayName": "setMaxBufferBehind"
+        },
+        {
+            "path": "./getMaxBufferBehind.md",
+            "displayName": "getMaxBufferBehind"
+        },
+        {
+            "path": "./setMaxBufferAhead.md",
+            "displayName": "setMaxBufferAhead"
+        },
+        {
+            "path": "./getMaxBufferAhead.md",
+            "displayName": "getMaxBufferAhead"
+        },
+        {
+            "path": "./getMaxVideoBufferSize.md",
+            "displayName": "getMaxVideoBufferSize"
+        },
+        {
+            "path": "./setMaxVideoBufferSize.md",
+            "displayName": "setMaxVideoBufferSize"
+        }
```

Comment:
I think it would be better to put the setter before the getter, as the API documentation can be read in order and the getter only really makes sense when you understand what the setter is about (at least in terms of how they are documented).